src/tests/request.test.js

```diff
@@ -0,0 +1,55 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import jwt from 'jsonwebtoken';
+import app from '../app';
+import mockData from './mockData';
+
+chai.use(chaiHttp);
+chai.should();
+
+const userSignUp = () => {
+  describe('View all my Requests.(GET) ', () => {
+    it('it should return 200 on successful signIn', (done) => {
```

Comment:
This test seems to be repeated on line 32!

Reply:
Yes, I repeated that login on view all requests as  I refer to the userId that I get from the token generated when a user logs in to make a search of all his requests. 

 Here is my search for viewing all the requests.

<img width="440" alt="Screen Shot 2020-02-22 at 13 44 46" src="https://user-images.githubusercontent.com/48021034/75091761-8ccefa80-5579-11ea-98a9-46d7fc9f2517.png">